packages/commonwealth/server/routes/verifyAddress.ts

```diff
@@ -286,7 +283,7 @@ const verifySignature = async (
     // both in base64 encoding
     const { signature: sigObj, publicKey } = JSON.parse(signatureString);
     isValid = nacl.sign.detached.verify(
-      Buffer.from(`${addressModel.verification_token}\n`),
+      Buffer.from(JSON.stringify(canvasMessage)),
```

Comment:
Is losing the newline here safe?

Reply:
From testing it, yes it works without the newline. Do you know why the newline was there in the first place?